services/web-app/next.config.js

```diff
@@ -29,20 +29,6 @@ module.exports = withMDX({
   images: {
     domains: ['raw.githubusercontent.com']
   },
-  sassOptions: {
-    includePaths: [path.join(__dirname, 'styles')],
-    prependData: `
-      @use '~@carbon/motion/scss/motion' as *;
-      @use '~@carbon/react/scss/breakpoint' as *;
-      @use '~@carbon/react/scss/spacing' as *;
-      @use '~@carbon/react/scss/theme' as *;
-      @use '~@carbon/react/scss/themes';
-      @use '~@carbon/react/scss/type' as *;
-      @use '~@carbon/react/scss/zone';
-      @use './styles/mixins' as *;
-      @use '~carbon-components/scss/globals/scss/_layout' as layout;
-    `
-  },
```

Comment:
> means we could benefit from codemods in the future if there's a v11 to v12 migration.

@mattrosno good question, in theory it would make codemods more "safe" but we could also just sniff for specific variable/function/mixin names. This whole area is pretty unexplored so 🤷 lol

Reply:
@joshblack ya, just thinking it'd be easier for a codemod to crawl import/use statements and then go from there. Instead of guessing names which could be non-Carbon. This is unrelated to why we're doing this, but calling out less global Sass could be good for us in the future.